docs/protocol-tutorial/0010-states-channels.md

```diff
@@ -0,0 +1,305 @@
+            participants: Address[];
+            channelNonce: Uint64;
+            appDefinition: Address;
+            challengeDuration: Uint48;
+        }
+
+        export interface VariablePart {
+            outcome: ExitFormat.Exit; // (2)
+            appData: Bytes;
+            turnNum: Uint48;
+            isFinal: boolean;
+        }
+    ```
+
+    1. `Bytes32`, `Bytes`, `Address`, `Uint256`, `Uint64` are aliases to the Javascript `string` type. They are respresented as hex strings. `Uint48` is aliased to a `number`.
+    2. This composite type is explained in the section on [outcomes](./0030-outcomes.md).
+
+=== "Go"
+
+    ```Go
+    import (
+        "github.com/statechannels/go-nitro/channel/state/outcome"
+        "github.com/statechannels/go-nitro/types" // (1)
+    )
+
+    type (
+        FixedPart struct {
+            ChainId           *types.Uint256
+            Participants      []types.Address
+            ChannelNonce      uint64
+            AppDefinition     types.Address
+            ChallengeDuration uint32
+        }
+
+        VariablePart struct {
+            AppData types.Bytes
+            Outcome outcome.Exit // (2)
+            TurnNum uint64
+            IsFinal bool
+        }
+    )
+    ```
+
+    1. `types.Address` is an alias to go-ethereum's [`common.Address`](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.8/common#Address) type. `types.Bytes32` is an alias to go-ethereum's [`common.Hash`](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.8/common#Hash) type.
+    2. This composite type is explained in the section on [outcomes](./0030-outcomes.md).
+
+!!! info
+
+    States are usually submitted to the blockchain as a single fixed part and multiple (signed) variable parts. This is known as a ["support proof"](#support-proofs).
+
+Let's take each property in turn:
+
+### Chain id
+
+This needs to match the id of the chain where assets are to be locked (i.e. the 'root' of the funding graph for this channel). In the event of a mismatch, the channel cannot be concluded and funds cannot be unlocked.
+
+### Participants
+
+This is a dynamic array of Ethereum addresses, each derived from an ECDSA private key in the usual manner. Each address represents a participant in the state channel who is able to commit to state updates and thereby cause the channel to finalize on chain.
+
+!!! warning
+
+    Before joining a state channel, you (or your off-chain software) should check that it has length at least 2, but no more than 255, and include a public key (account) that you control. Each entry should be a nonzero ethereum address.
+
+### ChannelNonce
+
+This is a unique number used to differentiate channels with an otherwise identical `FixedPart`. For example, if the same participants want to run the same kind of channel on the same chain as a previous channel, they can choose a new `ChannelNonce` to prevent state updates from from the existing channel being replayed.
```

Comment:
```suggestion
This is a unique number used to differentiate channels with an otherwise identical `FixedPart`. For example, if the same participants want to run the same kind of channel on the same chain as a previous channel, they can choose a new `ChannelNonce` to prevent state updates for the original channel from being replayed on the new one.
```